homeassistant/components/elevenlabs/tts.py

```diff
@@ -0,0 +1,112 @@
+"""Support for the ElevenLabs text-to-speech service."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from elevenlabs.client import AsyncElevenLabs
+from elevenlabs.core import ApiError
+from elevenlabs.types import Model, Voice as ElevenLabsVoice
+
+from homeassistant.components.tts import (
+    ATTR_VOICE,
+    TextToSpeechEntity,
+    TtsAudioType,
+    Voice,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import EleventLabsConfigEntry
+from .const import CONF_VOICE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: EleventLabsConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up ElevenLabs tts platform via config entry."""
+    client = config_entry.runtime_data.client
+    voices = (await client.voices.get_all()).voices
+    default_voice_id = config_entry.options.get(CONF_VOICE, voices[0].voice_id)
```

Comment:
```suggestion
    assert CONF_VOICE in config_entry.options and CONF_MODEL in config_entry.options
    default_voice_id = config_entry.options[CONF_VOICE]
```

Reply:
This is not needed right?

Reply:
The assert? Agreed.

Reply:
The change from get to keys neither, though with my new changes the CONF_VOICE key should always be set, so I would remove the `.get()` here in favour of __getitem__

Reply:
Yea that's awesome!